lib/data/replacements.json

```diff
@@ -764,6 +764,16 @@
       }
     ]
   },
+  "mvdan-sh-to-sh-syntax": {
+    "description": "`mvdan-sh` uses unmaintained `gopherjs` so it is very outdated, and now `mvdan-sh` is deprecated and `sh-syntax` is recommended by the original author.",
+    "packageRules": [
+      {
+        "matchDatasources": ["npm"],
+        "matchPackageNames": ["mvdan-sh"],
+        "replacementName": "sh-syntax"
+      }
+    ]
+  },
```

Comment:
We'd get rid of this change entirely, won't we?

Reply:
You'd put the package rule in lib/config/presets/internal/workarounds.ts lines 276-278.

Reply:
I don't know they're same thing previously. The typing is not great at `workaround.ts`.